conversation_simulator/editor/dialogger.js

```diff
@@ -1196,6 +1196,78 @@ function gameData () {
   return nodes
 }
 
+function validateGraph () {
```

Comment:
Let's do encapsulation here:

create a separate file Validator.js

Validator.getGraphErrors =function(graph) { ... }  //pass in graph object

Reply:
Ok done!